components/aiprompt/views/commands.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Commands View
+page_title: AIPrompt Commands View
+description: Explore the AIPrompt Commands View.
+slug: aiprompt-views-commands
+tags: telerik,blazor,aiprompt,ai,prompt,commands
+published: True
+position: 30
+---
+
+# AIPrompt Commands View
+
+The Commands View displays a set of predefined commands the user can browse and optionally execute. The commands are passed to the component through the `Commands` parameter, which expects a collection of type `List<AIPromptCommandDescriptor>`. Commands can also be organized through a hierarchy, through a parent-child relationship.
+
+>note Commands can be organized through a hierarchy, with a parent-child relationship. Parent commands cannot be directly executed, and only one level of nesting is supported.
+
+## AIPromptCommandDescriptor Parameters
+
+The following properties enable you to customize each command:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type | Description |
+| ----------- | ----------- | ----------- |
+| `Id` | `string` | Defines the Id of the command.
```

Comment:
We normally skip "Defines" or similar phrases, except for boolean paramters, where it's hard to achieve it.
```suggestion
| `Id` | `string` | The Id of the command.
```